packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

```diff
@@ -83,14 +83,14 @@ Some routes require authentication to render. Defining these protected routes is
 
 ### `/ - index page`
 
-First, update `src/app/app.component.html` to provide the Login logic:
+First, update `src/app/app.component.html` to provide the Login logic, replacing `${widgetVersion}` with the [latest version](https://github.com/okta/okta-signin-widget/releases/) of the Widget (-=OKTA_REPLACE_WITH_WIDGET_VERSION=-):
 
```

Comment:
Does this mean, the doc will always say latest version of widget is necessary?  I was hoping it would say 6.3.5, since that is from where the magic link feature's latest bug was fixed

Reply:
Understood from offline conversation that it will be latest widget version for some time.  After the feature is mature, we will change this to be a minimum version required.